README.md

```diff
@@ -1,6 +1,6 @@
 # Barefoot Nomad - Making company travel and accommodation easy and convenient
 
-[![Reviewed by Hound](https://img.shields.io/badge/Reviewed_by-Hound-8E64B0.svg)](https://houndci.com)  [![Build Status](https://travis-ci.com/andela/phoenix-bn-backend.svg?branch=develop)](https://travis-ci.com/andela/phoenix-bn-backend) [![Maintainability](https://api.codeclimate.com/v1/badges/28351a9e117de0ecbba0/maintainability)](https://codeclimate.com/github/andela/phoenix-bn-backend/maintainability) [![Coverage Status](https://coveralls.io/repos/github/andela/phoenix-bn-backend/badge.svg?branch=develop)](https://coveralls.io/github/andela/phoenix-bn-backend?branch=develop)
+[![Reviewed by Hound](https://img.shields.io/badge/Reviewed_by-Hound-8E64B0.svg)](https://houndci.com)  [![Build Status](https://travis-ci.org/andela/phoenix-bn-backend.svg?branch=develop)](https://travis-ci.org/andela/phoenix-bn-backend) [![Maintainability](https://api.codeclimate.com/v1/badges/28351a9e117de0ecbba0/maintainability)](https://codeclimate.com/github/andela/phoenix-bn-backend/maintainability) [![Test Coverage](https://api.codeclimate.com/v1/badges/28351a9e117de0ecbba0/test_coverage)](https://codeclimate.com/github/andela/phoenix-bn-backend/test_coverage)
 
```

Comment:
Why are you making this change here?

Reply:
I can see that you replaced the coveralls badge with codeclimate badge

Reply:
That's my error. I've reverted that.

Reply:
The `checkifUserExistBeforeLogin` checks if user exist during login and gets user email from req.body.
However, the `confirmUserExists` checks if user is valid during registration. It gets user email  from token sent to the user. As user should not be allowed to update his email during registration.

The two middleware are not exactly the same. Please advise.